spec/type_aliases/upstreamzone_spec.rb

```diff
@@ -1,7 +1,6 @@
 require 'spec_helper'
 
 describe 'Nginx::UpstreamZone' do
-  it { is_expected.to compile.with_all_deps }
```

Comment:
This wasn't valid here and failed. (Which is how I know for sure that these weren't being run)